aarch64 binaries support

bucanero opened the pull request:
Binaries built under debian 12 arm64. 
Tested on Debian 12 Arm64, and Ubuntu 22 Arm64.

Adding arm64 could solve issues like #79 . Also this can help people on arm64 macs (M1/M2/etc) to run the scripts on a virtual machine.

Sources used:
- https://github.com/AKuHAK/pfsshell/tree/8Mb
- https://github.com/AKuHAK/hdl-dump/tree/8M
- https://github.com/ps2homebrew/pfsshell
- https://github.com/exfatprogs/exfatprogs/archive/refs/tags/1.2.2.zip
- https://repository.timesys.com/buildsources/s/sqlite/sqlite-2.8.17/sqlite-2.8.17.tar.gz
- https://github.com/bucanero/save-decrypters/tree/master/ps2-apa-header-checksum
- https://github.com/bucanero/psv-save-converter
- https://github.com/bucanero/pops2cue/

CosmicScale commented:
Fully tested Installing PSBBN, the game installer and music installer on a Raspberry Pi. Everything works great. I'm in the middle of writing a big revision at the moment but I'll add arm64 support with the next big update. Thanks for compiling them :)

CosmicScale commented:
Yep, that would be great!

bucanero commented:
> Yep, that would be great!

all done 🎉  original files restored, and new binaries moved to `scripts/helper/aarch64/` 👍 

Btw, I suggest doing  a "Squash and merge" to keep things clean as I was originally replacing the files and then I moved them.

CosmicScale commented:
Could you please also compile [cue2pops](https://github.com/israpps/cue2pops) for ARM64? Also remove `PSBBN-Definitive-Patch.sh`. Thanks!

CosmicScale commented:
> I'd suggest using the newer [cue2pops v2.3](https://github.com/bucanero/pops2cue/blob/main/cue2pops.c); the newer version was improved and removed all those special parameters `gap++, gap--, vmode, trainer, etc`, it will just convert ps1 cue/bin to POPS automatically without any additional input or customizations.

That sounds good, I wasn't aware of that version :)

> you mean remove the whole file from my PR branch? It will delete your file when merged.

Changes have been made to  `PSBBN-Definitive-Patch.sh` since you raised this PR, if I merge, I think it will overwrite them.

bucanero commented:
> That sounds good, I wasn't aware of that version :)
> 
> Changes have been made to `PSBBN-Definitive-Patch.sh` since you raised this PR, if I merge, I think it will overwrite them.

done, I've added `cue2pops` binary (v2.3) , built on Debian 12 ARM64 👍 
Also cleaned and removed changes to the PSBBN script, so this PR only adds the new binaries to the `aarch64` folder

note, use "squash & merge" if you want 1 single clean commit with all the new binaries in your main branch.
